public/app/features/alerting/unified/components/receivers/ReceiversSection.tsx

```diff
@@ -33,7 +33,9 @@ export const ReceiversSection: FC<Props> = ({
         </div>
         {showButton && (
           <Link to={addButtonTo}>
-            <Button icon="plus">{addButtonLabel}</Button>
+            <Button type="submit" icon="plus">
```

Comment:
This one should be a `type="button"`, I see it's also nested in a `<a>` tag, which is illegal :/

Reply:
good catch @gillesdemey 🙌🏾, I will double-check again these changes